<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsrMainMenu</class>
 <widget class="QMainWindow" name="UsrMainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>冬奥纪念品交易平台-用户主菜单</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buyer_btn">
       <property name="text">
        <string>我是买家</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seller_btn">
       <property name="text">
        <string>我是卖家</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="info_manage_btn">
       <property name="text">
        <string>个人信息管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_btn">
       <property name="text">
        <string>注销登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
